### PR TITLE
docs(common): macos build update

### PR DESCRIPTION
**Pull request by mcdurdin**
@keymanapp-test-bot skip

### CODE REVIEW

File: docs/build/macos.md [hunk 1]
```diff
@@ -20,19 +23,24 @@ The following projects **cannot** be built on macOS:
...
+versions does not build for arm64 (M1). The build will still work on earlier
+versions, but the installer won't be able to run on M1 Macs that do not have
+Rosetta 2 installed.
 
 ## Prerequisites
 
 Many dependencies are only required for specific projects.
 
-* XCode (iOS, macOS) 12.4 or later
+* XCode (iOS, macOS) 12.4 or later is needed only for Keyman for Mac and Keyman
```
Comment (darcywong00) [inline docs/build/macos.md:42]:
```suggestion
* XCode (iOS, macOS) 12.4 or later is needed only for Keyman for macOS and Keyman
```

according to keyman.com/mac :smile:

File: docs/build/macos.md [hunk 2]
```diff
@@ -48,73 +56,83 @@ These dependencies are also listed below if you'd prefer to install manually.
...
-  ```
+## Keyman for Android Dependencies
 
-* sentry-cli (optional)
-  - Uploading symbols for Sentry-based error reporting
+* openjdk 8, Android SDK, Android Studio, Ant, Gradle, Maven
 
-  ```
+```shell
+brew install openjdk@8 android-sdk android-studio ant gradle maven
```
Comment (darcywong00) [inline docs/build/macos.md:112]: Don't we use openjdk@11 on master? (l.109 too)

**Review by darcywong00 — APPROVED** [review]
lgtm - minor nits

File: docs/build/macos.md (hunk 2 above)
Comment (mcdurdin) [reply docs/build/macos.md:112]:
```suggestion
brew install openjdk@11 android-sdk android-studio ant gradle maven
```

File: docs/build/macos.md [hunk 3]
```diff
@@ -48,73 +56,83 @@ These dependencies are also listed below if you'd prefer to install manually.
...
 
-  ```bash
-  brew tap homebrew/cask-versions
-  brew install --cask --no-quarantine wine-stable
-  ```
+## Keyman for Android Dependencies
 
-* sentry-cli (optional)
-  - Uploading symbols for Sentry-based error reporting
+* openjdk 8, Android SDK, Android Studio, Ant, Gradle, Maven
```
Comment (mcdurdin) [inline docs/build/macos.md:109]:
```suggestion
* openjdk 11, Android SDK, Android Studio, Ant, Gradle, Maven
```

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.196-alpha](https://keyman.com/downloads/releases/alpha/17.0.196)

**Pull request merged**